content/en/docs/reference/node/kubelet-files.md

```diff
@@ -0,0 +1,132 @@
+---
+content_type: "reference"
+title: Local Files And Paths Used By The Kubelet
+weight: 42
+---
+
+The {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} is mostly a stateless
+process running on a Kubernetes {{< glossary_tooltip text="node" term_id="node" >}}.
+This document outlines files that kubelet reads and writes.
+
+The kubelet typically uses the {{< glossary_tooltip text="control plane" term_id="control-plane" >}} as
+the source of truth on what needs to run on the Node, and the
+{{<glossary_tooltip text="container runtime" term_id="container-runtime">}} to retrieve
+the current state of containers. So long as you provide a _kubeconfig_ (API client configuration)
+to the kubelet, the kubelet does connect to your control plane; otherwise the node operates in
+_standalone mode_.
+
+On Linux the kubelet also relies on reading cgroups and various system files to collect metrics. 
+
+There are also a few other files that are used by the kubelet as well as kubelet communicates using local Unix-domain sockets. Some are sockets that the
+kubelet listens on, and for other sockets the kubelet discovers them and then connects
+as a client.
+
+
+## Configuration
+
+### Kubelet configuration files
+
+The path to the kubelet configuration file can be configured
+using the command line argument `--config`. kubelet also supports
+[drop-in configuration files](/docs/tasks/administer-cluster/kubelet-config-file/#kubelet-conf-d)
+to enhance configuration.
+
+### Certificates
+
+Certificates are typically located at `/var/lib/kubelet/pki`,
+but can be configured using the `--cert-dir` kubelet command line argument.
+Names of certificate files are also configurable.
+
+### Systemd unit settings
+
+When kubelet is running as a systemd unit, some kubelet configuration may be declared
+in systemd unit settings file. Typically it includes:
+
+- command line arguments to [run kubelet](/docs/reference/command-line-tools-reference/kubelet/)
+- environment variables, used by kubelet or [configuring golang runtime](https://pkg.go.dev/runtime#hdr-Environment_Variables)
+
+## State
+
+### Manifests
+
+Manifests for static pods are typically located in `/etc/kubernetes/manifests`.
+Location can be configured using the `staticPodPath` kubelet configuration option.
```

Comment:
IMO, these are desired state (ie, configuration) rather than actual state.

Reply:
moved. I was debating with myself and don't feel strongly